recipes/pispy/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  name: pispy-client
+  version: "0.9.0"
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name | replace("-", "_") }}/${{ name | replace("-", "_") }}-${{ version }}.tar.gz
+  sha256: dff96b598412c6d051a173550a532c39180da42b389376ac38e0a517584a1faa
+
+build:
+  noarch: python
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  python:
+    entry_points:
+      pispy = pispy.__main__:run
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - hatchling
+  run:
+    - python >=${{ python_min }}
+    - httpx
+    - packaging
+    - textual >=0.68.0
+
+tests:
```

Comment:
I think it would be worth it to put in a `python` check, which would also get us a `pip check`